actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -137,6 +138,7 @@ def log_error(request, wrapper)
         return unless logger
 
         exception = wrapper.exception
+        return if !log_rescued_responses(request) && ActionDispatch::ExceptionWrapper.rescue_responses.key?(exception.class.name)
```

Comment:
Let's move the `ActionDispatch::ExceptionWrapper.rescue_responses.key?(exception.class.name)` check to the wrapper.

Reply:
Done.